server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckIn.java

```diff
@@ -32,7 +31,6 @@ public CheckIns(UUID teamMemberId, UUID pdlId, Date checkInDate, String targetQt
     @Id
     @Column(name="uuid")
     @NotNull
-    @GeneratedValue    
     private UUID pdlId;
```

Comment:
This column name should be "pdlId" and it shouldn't be the @Id.  This class needs a...
```
@Id
@Column(name="uuid")
@NotNull
@GeneratedValue
private UUID id;
```
...as well.